From 977b8eb535eb89afdad5e36e30c180b640c8a7a6 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 19 Jun 2024 23:47:04 +0800
Subject: [PATCH] fix(CMakeLists.txt): do not strip during install time

AOSC OS splits out the debug symbols into a -dbg package. Stripping during
install time prevents this from being done.
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 57d82eff..0d01fd96 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,10 +78,6 @@ set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Og")
 string(REPLACE "-O2" "-O3" CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}")
 string(REPLACE "-O2" "-O3" CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE}")
 
-if (NOT APPLE)
-    set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE} -s")
-endif()
-
 if (WIN32)
     option(BUILD_STATIC "Statically link dependencies" OFF)
 endif()
-- 
2.45.2

